intellij-setup.md

```diff
@@ -28,23 +28,27 @@ Android Studio (and various other JetBrains editors) is currently not supported.
 
 ### Installing the plugins
 
-We provide two plugins:
+To use IntelliJ with Flutter, you must install two plugins:
```

Comment:
Actually, we don't mention that below (we just say the SDK is bundled), and I was not aware. When testing I installed the Dart plugin, then the Flutter plugin, as the instructions said to do - so didn't see the new behavior.